contracts/CO30_Matias_ComeFlyWithMe.pja307/do_not_edit/initServer.sqf

```diff
@@ -0,0 +1,25 @@
+// Calculate mission start
```

Comment:
Not using latest from template

Reply:
Never got the memo that the template got updated